src/status_im/contexts/wallet/connected_dapps/view.cljs

```diff
@@ -83,7 +83,8 @@
   []
   (let [{:keys [bottom]}                   (safe-area/get-insets)
         {:keys [color] :as wallet-account} (rf/sub [:wallet/current-viewing-account])
-        sessions                           (rf/sub [:wallet-connect/persisted-sessions])
+        sessions                           (rf/sub
+                                            [:wallet-connect/persisted-sessions-for-current-account])
```

Comment:
we shouldn't rely on the persisted sessions by default, but rather on the ones coming from web3wallet, as they will be most up-to-date. We should just use the persisted ones in cases when the user has no internet connection and keep it synced with the state from wallet-connect active sessions.

Reply:
cc: @stefandunca we are approaching it the same way in desktop, right?